### DISCUSSION TITLE
`v-select`: custom style for every item

### DISCUSSION THREAD

**Discussion by JakobMiksch** [vuetifyjs/vuetify#17855] [Q&A] [answered]
I have a dropdown menu with [`v-select`](https://vuetifyjs.com/en/components/selects/). I would like to give every item in the list a background color depending on its properties. 

I already discovered the slot named `item`, but it seems to replace the whole item.  However, I would just like to edit the appearance of the items that will be rendered in the list.

Does anyone have an idea how to accomplish that?

Thanks in advance.

**Comment by KaelWD** [comment, marked as answer]
```html
<template #item="{ props }">
  <v-list-item v-bind="props" />
</template>
```
then do whatever else you want to it

**Reply by JakobMiksch** [reply]
Great. I will try that! Thanks already.

**Reply by JakobMiksch** [reply]
Thanks to @KaelWD 's answer 🥇  I could solve it quickly 
Here an extended snippet how to change the color of the selected item.

```vue
<v-select
  :items="myItems"
>
  <template #item="{ props }">
    <v-list-item
      v-bind="props"
      :base-color="props.value === 'my_special_value' : 'red' : 'green'"
    />
  </template>
</v-select>
```
